posts/viewsfolder/feed_views.py

```diff
@@ -17,6 +17,12 @@
 
 
 class FrontEndPublicPosts(TemplateView):
+    def get_posts(self):
+        try:
```

Comment:
Use the helper for getting local post.

Reply:
For public feed too?

Reply:
We can look at that later but this was removed and we shouldn't add it back because we need to also get external posts which we have

Reply:
Okay I just took this our for now as I am confused

Reply:
Sorry I misread where this was. It is fone